### PR TITLE
enhancement/login-page-mui

### PR DESCRIPTION
**Pull request by yablonya**

### CODE REVIEW

File: src/components/pages/login-page/LoginPage.styles.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    maxWidth: '480px',
+    width: '100%',
+    alignItems: 'flex-start',
+  },
+};
+
+export const divider: SxProps<Theme> = {
+  height: 'inherit',
+  width: '2px',
+  borderColor: '#FAFAFA',
```
Comment (Markusplay) [inline src/components/pages/login-page/LoginPage.styles.ts:39]: use palette

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  [theme.breakpoints.down('desktopSemiMedium')]: {
+    display: 'none',
+  },
+};
+
+export const divider: SxProps<Theme> = {
+  width: '100%',
+  '&::before': {
+    borderColor: '#FAFAFA',
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts:27]: use palette

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  '&::after': {
+    borderColor: '#FAFAFA',
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts:30]: use palette

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  },
+};
+
+export const mobileText: SxProps<Theme> = {
+  fontSize: '16px',
+  lineHeight: '1.5',
+  fontWeight: 500,
+  display: 'none',
+  margin: '28px 0 14px 0',
+  color: '#D4D4D4',
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts:40]: use palette

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  '&::after': {
+    borderColor: '#FAFAFA',
+  },
+};
+
+export const mobileText: SxProps<Theme> = {
+  fontSize: '16px',
+  lineHeight: '1.5',
+  fontWeight: 500,
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts:37]: use typography

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.tsx
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.tsx:1]: use new `mui` Button component

File: src/components/pages/login-page/components/logo-register-block/LogoRegisterBlock.tsx [hunk 6]
```diff
@@ -1,36 +1,39 @@
...
+          style={{ margin: 0, padding: '8px' }}
           src="/images/login-page/new-logo.png"
           alt="fict advisor logo"
           priority
           width={300}
           height={54}
         />
       </Link>
-      <h3 className={styles['register-text']}>Ти ще не з нами? Приєднуйся!</h3>
+      <Typography variant="h3" sx={styles.registerText}>
```
Comment (Markusplay) [inline src/components/pages/login-page/components/logo-register-block/LogoRegisterBlock.tsx:25]: as your font doesn't changes with width size, remove styles for fonts in `registerText` and set correct `variant` for `Typography`

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.tsx [hunk 7]
```diff
@@ -43,34 +44,30 @@ const RightBlock = () => {
...
         onClick={handleClick}
         className={styles['telegram-button']}
       />
-      <Divider text="або" textAlign={DividerTextAlign.CENTER} />
+      <Divider textAlign="center" sx={sxStyles.divider}>
+        або
+      </Divider>
       <LoginForm />
-      <p className={styles['mobile-text']}>Ти ще не з нами? </p>
+      <Typography sx={sxStyles.mobileText}>Ти ще не з нами? </Typography>
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.tsx:70]: you say `mobileText` but it exist even on dekstop. Rename style

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.tsx [hunk 8]
```diff
@@ -43,34 +44,30 @@ const RightBlock = () => {
           fill
         />
       </Link>
-      <h3 className={styles['register-header']}>З поверненням!</h3>
+      <Typography variant="h3" sx={sxStyles.loginHeader}>
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.tsx:47]: as your font doesn't changes with width size, remove styles for fonts in `loginHeader` and set correct `variant` for `Typography`

File: src/components/pages/login-page/components/logo-register-block/LogoRegisterBlock.tsx
Comment (Markusplay) [inline src/components/pages/login-page/components/logo-register-block/LogoRegisterBlock.tsx:1]: use new `mui` Button component

File: src/components/pages/login-page/components/login-form/LoginForm.tsx
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form/LoginForm.tsx:1]: you removed styles for input, wtf ?

File: src/components/pages/login-page/LoginPage.styles.ts [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+  alignItems: 'center',
+  backgroundSize: '100% 100%',
+  padding: '0 30px',
+
+  [theme.breakpoints.down('desktopSemiMedium')]: {
+    backgroundSize: '100% 100%',
+  },
+};
+
+export const loginPage__content: SxProps<Theme> = {
```
Comment (Markusplay) [inline src/components/pages/login-page/LoginPage.styles.ts:20]: we write styles in camelCase

File: src/components/pages/login-page/LoginPage.styles.ts [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+  zIndex: 1,
+  display: 'flex',
+  flexDirection: 'row',
```
Comment (Markusplay) [inline src/components/pages/login-page/LoginPage.styles.ts:23]: flexDirection is row by default

**Review by Markusplay — CHANGES_REQUESTED** [review]

File: src/components/pages/login-page/LoginPage.styles.ts [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+  width: '100%',
+  minHeight: '100vh',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  backgroundSize: '100% 100%',
+  padding: '0 30px',
+
+  [theme.breakpoints.down('desktopSemiMedium')]: {
+    backgroundSize: '100% 100%',
```
Comment (Markusplay) [inline src/components/pages/login-page/LoginPage.styles.ts:16]: duplicate

File: src/components/pages/login-page/LoginPage.styles.ts [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+  zIndex: 1,
+  display: 'flex',
+  width: '100%',
+  minHeight: '635px',
+  maxWidth: '1200px',
+  justifyContent: 'space-around',
+
+  [theme.breakpoints.down('desktopSemiMedium')]: {
+    maxWidth: '480px',
+    width: '100%',
```
Comment (Markusplay) [inline src/components/pages/login-page/LoginPage.styles.ts:30]: duplicate

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+};
+
+export const divider: SxProps<Theme> = {
+  width: '100%',
+  '&::before': {
+    borderColor: theme.palette.grey[800],
+  },
+  '&::after': {
+    borderColor: theme.palette.grey[800],
+  },
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.styles.ts:69]: duplicate

**Review by Markusplay — CHANGES_REQUESTED** [review]

File: src/components/pages/login-page/components/login-form-block/LoginFormBlock.module.scss [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    display: flex;
+  }
+}
+
+@media screen and (max-width: $breakpoint-mobile-medium) {
+  .login-button{
+    padding: 12px 24px;
+  }
+}
+
```
Comment (Markusplay) [inline src/components/pages/login-page/components/login-form-block/LoginFormBlock.module.scss:33]: do you use this styles anywhere ?

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Review by Markusplay — APPROVED** [review]

**Review by igaryakqwe — APPROVED** [review]

**Pull request merged**